modules/local/concatblocks.nf

```diff
@@ -0,0 +1,26 @@
+process CONCATBLOCKS {
+    tag "${meta.id}.bed"
+    label "process_medium"
+
+    conda "conda-forge::coreutils=9.1"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+            'https://depot.galaxyproject.org/singularity/ubuntu:20.04' :
+            'ubuntu:20.04' }"
+
+    input:
+    tuple val(meta), path(mergeblocks)
+
+    output:
+    tuple val(meta), path("*.bed"), emit: chainfile
+    path "versions.yml", emit: versions
+
+    script:
```

Comment:
I'm surprised `nf-core lint` didn't flag this ! It would be just a warning anyway. Just adding it now to future-proof the module
```suggestion
    when:
    task.ext.when == null || task.ext.when

    script:
```